torch/_inductor/codegen/triton_utils.py

```diff
@@ -58,11 +70,17 @@ def signature_to_meta(
     *,
     size_dtype: str,
     indices: Optional[List[int]] = None,
+    zero_dim_cpu_tensor_list: Optional[Set[str]] = None,
 ) -> Dict[int, str]:
+    call_args = [arg.buffer for arg in signature if isinstance(arg, TensorArg)]
```

Comment:
will remove in next commit

Reply:
fixed in https://github.com/pytorch/pytorch/pull/132702/commits/502ed996db5172e5bfec8851d5f3d1bd2681a395